src/controllers/auth.js

```diff
@@ -19,9 +19,15 @@ const saveUserToTokenAndCookie = (req, res) => {
     maxAge: 1000 * 60 * 60 * 24 * 14,
   });
 
-  res.redirect('/');
+  res.redirect('/api/auth/me');
```

Comment:
this should be ("/api/auth/profile). You don't need to write code for it, I already created that endpoint